src/main/java/com/vinylteam/vinyl/util/PropertiesReader.java

```diff
@@ -35,11 +34,11 @@ public PropertiesReader() {
         String env = System.getenv("env");
         logger.debug("Created and initialised String env with the value " +
                 "of environmental variable \"env\" {'env':{}}", env);
-        if (env == null) {
+        if (System.getenv("env") == null) {
             dependentOnEnvPropertiesFile = "dev.application.properties";
 
             try (InputStream inputStream = getClass().getClassLoader()
-                    .getResourceAsStream(dependentOnEnvPropertiesFile)) {
+                    .getResourceAsStream("dev.application.properties")) {
```

Comment:
I think you are using the version that accidentally changed PropertiesReader to how it was before logger.

Do you use String env?